roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -40,6 +40,9 @@
     openshift_hosted_registry_images: "{{ openshift.hosted.registry.registryurl | default('openshift3/ose-${component}:${version}')}}"
     openshift_hosted_registry_volumes: []
     openshift_hosted_registry_env_vars: {}
+    openshift_hosted_registry_route_certificates: "{{ openshift.hosted.registry.route.certificates | default(false) }}"
+    openshift_hosted_registry_route_host: "{{ openshift.hosted.registry.route.host | default(false) }}"
+    openshift_hosted_registry_route_termination: "{{ openshift.hosted.registry.route.termination | default('passthrough') }}"
```

Comment:
@detiber @abutcher @mtnbikenc 

I'm not sure if moving these to defaults will result in the correct behavior.  I need the variable to default to the openshift.hosted.registry.route.* variable and if that does not exist then default to a different value.

The other way would be to set_fact and set it to the desired when it is defined.  We get that same behavior here I believe.  In the next version we want to refactor how this works so I think variables getting passed in with their proper settings would greatly help.